### DISCUSSION TITLE
WalletConnect

### DISCUSSION THREAD

**Discussion by FrontlineHeroesToken** [hashlips-lab/nft-erc721-collection#238] [Q&A]
Is there a thread to add wallet connect for other mobile devices? "if not" is there a way?

**Comment by mpixelz** [comment]
any luck?

**Comment by liarco** [comment]
Hi, this features is not available for version `2.x`, but it's planned for version `3.x` which is the next major release. There is no ETA for it at the moment, but at least a public beta may come in 2/3 weeks.

**Reply by mpixelz** [reply]
Thanks for the update @liarco ..

just a quick question: i already have a contract deployed using v1.. would it be possible to use minting dapp from v2 or if there are any steps to update? and if we would be able to update to v3 when it comes out?

much appreciated. Thanks.
